plugins/drng/drng.go

```diff
@@ -14,6 +18,43 @@ var (
 	ErrParsingCommitteeMember = errors.New("cannot parse committee member")
 )
 
+func configureDRNG() *drng.DRNG {
+	// parse identities of the committee members
+	committeeMembers, err := parseCommitteeMembers()
+	if err != nil {
+		log.Warnf("Invalid %s: %s", CfgDRNGCommitteeMembers, err)
```

Comment:
`configureDRNG` uses `log`, but the logger is not initialized in `Instance()`. This will fail if `Instance()` is called before `configure`.